### ISSUE TITLE
Wrong URL for api/user after successful auth.

### ISSUE THREAD

**Issue by vlitvin** [ansible-semaphore/semaphore#628]
Semaphore v2.6.1 Wrong URL for `api/user` after successful auth.

Steps to reproduce:
- Install Semaphore as described in documentation.
- Open <web_host> URL in browser
- As <web_host>/api/user returns 401 (which is expected, i guess), we are redirected to <web_host>/auth/login
- Submit <user> and <password> from setup step.

Expected behavior:
- Login screen reports successful auth.
- Open main page as authenticated user.

Observed behavior:
- Login screen reports successful auth.
- Browser redirected back to `<web_host>/auth/login`
- No related logs in semaphore daemon output
- In browser console log:
  ```
  GET https://api/user/ net::ERR_NAME_NOT_RESOLVED
  ```
  Which is clearly missing `<host>/` part.
- In net log:
  ```
  Request:
    Request URL: https://<host>/api/user
    Request Method: GET
    ...
  Response:
    Status Code: 301 Moved Permanently
    Location: https:///api/user/
  ```
  Thats explains error above.
- Moreover, dumping traffic between nginx and semaphore confirms thats semaphore returns 301.

**Comment by fiftin** [comment]
I can not reproduce. What OS dot you use? Looks like problem with configuration. Can you please give your config.json?

**Comment by fiftin** [comment]
I fixed bug with routing, fix available in new release: https://github.com/ansible-semaphore/semaphore/releases/tag/v2.6.2

**Comment by andreadragotta** [comment]
I have the same problem with 2.6.2 version

**Comment by fiftin** [comment]
@andreadragotta what OS do you use? Could you share your `config.json`?

**Comment by andreadragotta** [comment]
OS: Centos 7

config.json
```

{
 	"mysql": {
 		"host": ".....:3306",
 		"user": "semaphore",
 		"pass": "....",
 		"name": "semaphore"
 	},
 	"port": "",
 	"interface": "",
 	"tmp_path": "/tmp/semaphore",
 	"cookie_hash": "...",
 	"cookie_encryption": "....",
 	"email_sender": "",
 	"email_host": "",
 	"email_port": "",
 	"web_host": "http://ansiblesemaphore.myurl.com/",
 	"ldap_binddn": "",
 	"ldap_bindpassword": "",
 	"ldap_server": "",
 	"ldap_searchdn": "",
 	"ldap_searchfilter": "",
 	"ldap_mappings": {
 		"dn": "",
 		"mail": "",
 		"uid": "",
 		"cn": ""
 	},
 	"telegram_chat": "",
 	"telegram_token": "",
 	"concurrency_mode": "",
 	"max_parallel_tasks": 0,
 	"email_alert": false,
 	"telegram_alert": false,
 	"ldap_enable": false,
 	"ldap_needtls": false,
 	"old_frontend": false
 }
```

**Comment by fiftin** [comment]
@andreadragotta If you used v2.6.1 before, It can be browser cache. Unfortunately cache fo "301 Moved Permanently" hard to clear. Cloud you test in Guest mode (if you use Chrome)?

**Comment by andreadragotta** [comment]
No, i've installed the 2.6.2 for the first time and i've test with different browser with Guest mode.

I use nginx proxy pass with /etc/hosts config line for mapping.

**Comment by fiftin** [comment]
@andreadragotta Thank you. I can't reproduce the bug on Ubuntu and it is problem.

 But looks like I got where the bug. Could you try https://github.com/ansible-semaphore/semaphore/releases/tag/v2.6.3 ?

**Comment by andreadragotta** [comment]
Ok! It's work 😄 

Thank you

P.S. I've try to updgrade with semaphore -upgrade but the command show this:

```
current release is vv2.6.2
panic: cannot find binary for your system

goroutine 1 [running]:
main.main()
	/go/src/github.com/ansible-semaphore/semaphore/cli/main.go:32 +0x81f
```

**Comment by fiftin** [comment]
@andreadragotta Thank you! 

Yes, this is known bug: https://github.com/ansible-semaphore/semaphore/issues/565. Fix is planned for version 2.7. Currently I want to fix all UI bugs after migration to Vue.js.

**Comment by andreadragotta** [comment]
Perfect! 😄
